SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -119,6 +119,12 @@ void UGlobalStateManager::UpdateSingletonEntityId(const FString& ClassName, cons
 {
 	SingletonNameToEntityId[ClassName] = SingletonEntityId;
 
+	if (!NetDriver->StaticComponentView->HasAuthority(SpatialConstants::GLOBAL_STATE_MANAGER, SpatialConstants::GLOBAL_STATE_MANAGER_COMPONENT_ID))
```

Comment:
Need to note that this will have to changed in the server-travel branch since the constant ID for this no longer exists and there are now variables to indicate authority.